chapters/hpmor-chapter-013.tex

```diff
@@ -160,10 +160,8 @@ \chapter{Asking the Wrong Questions}
 “How can I earn more points?” Harry said out loud.
 
 Then he pulled out the drawer.
-\begin{align*}\intertext{\scshape \centering
-Opportunities to do good are everywhere\\
-but darkness is where the light needs to be}
-\hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
+\begin{align*}\intertext{\scshape \centering Opportunities to do good are everywhere\\
```

Comment:
Please undo this one, the markup's much easier to read with the newlines, and it also gives a better impression of the text.

Reply:
good catch, fixed